src/utils/hooks/useContext.ts

```diff
@@ -0,0 +1,9 @@
+import { useContext } from 'react';
+
+export default function useStarContainerContext<Type>(
+  ContextValue: React.Context<Type | null>,
+) {
+  const state = useContext(ContextValue);
+  if (!state) throw new Error('Provider not found');
```

Comment:
엄격히 분기문을 처리하는 것이 좋아보여요

```suggestion
  if (state == null) {
    throw new Error('ContextValue.Provider 가 존재하지 않습니다.');
  }
```